examples/vault-variable-test/runtime_config.toml

```diff
@@ -0,0 +1,5 @@
+[[config_provider]]
```

Comment:
Is it worth adding a comment to this file?  Or talk about it in a README?  Not sure, just a vague wonder...